contracts/PetPark.sol

```diff
@@ -1,7 +1,201 @@
-//SPDX-License-Identifier: Unlicense
+// SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.0;
 
+contract PetPark
+{
+    // -- SCAFFOLDING -------------------------
+    enum AnimalType
+    {
+        NONE
```

Comment:
It will be good to use a solidity plugin to format here - https://marketplace.visualstudio.com/items?itemName=NomicFoundation.hardhat-solidity

Reply:
Generally, all projects in production use formatted code. Although the code still works but it is better to have it formatted